### DISCUSSION TITLE
0.12 release?

### DISCUSSION THREAD

**Discussion by raybellwaves** [anitagraser/movingpandas#254] [General]
@anitagraser would you be so kind to cut a release please? I'm hoping to use the acceleration method. We can also try the release GitHub action once you create a tag. We can also see if the whl is installable via pyodide for your dashboard work

**Comment by anitagraser** [comment]
Hi Ray, I'll get to it as soon as I can. If you have some time, it would help if you could write the summary for the release, similar to https://github.com/anitagraser/movingpandas/releases/tag/v0.10

**Reply by raybellwaves** [reply]
❯ git log $(git describe --tags --abbrev=0)..HEAD --oneline
061035a (HEAD -> master, origin/master, origin/HEAD) added a `add_acceleration` method (#253)
f4b35ff Create release_to_pypi.yml (#240)
28595de Fixed typo in mrr_diagonal, fixes #249
81ebf4f _get_df_with_timedelta refactor (#248)
62154e9 Link to correct ipynb
8aa225c Add Twitter follow badge
f429698 Update README.md
cb159bf Add smoothing and cleaning example

**Reply by raybellwaves** [reply]
New features:

https://github.com/anitagraser/movingpandas/issues/253 New add_acceleration method

Includes fixes and enhancements for:

https://github.com/anitagraser/movingpandas/issues/249 Fix typo in mrr_diagonal

Behind the scenes:

[Added a link to a notebook for Trajectory cleaning & smoothing](https://github.com/anitagraser/movingpandas/commit/cb159bfd89895006d5212c73ab6e2439fdc92796)
https://github.com/anitagraser/movingpandas/issues/240 Use a GitHub action to release new versions.
[Created a twitter account](https://twitter.com/MovingPandasOrg)

**Comment by anitagraser** [comment]
Ok, done: https://github.com/anitagraser/movingpandas/releases/tag/v0.12

Let's see how that action is doing.

**Reply by anitagraser** [reply]
It failed, but there is something at https://pypi.org/project/movingpandas/0.12rc1

**Reply by anitagraser** [reply]
Ah, "Create Github Release" failed, because I just manually created this release ... yeah, the error makes sense. Why is this step there?

**Reply by raybellwaves** [reply]
Good question. Maybe there is a better way to use it. I'll have to see how they use it in geopandas

**Reply by raybellwaves** [reply]
https://github.com/geopandas/geopandas/actions/workflows/release_to_pypi.yml still not immediately sure how to use it
